dbutils/insert.go

```diff
@@ -70,6 +74,11 @@ func main() {
 		fmt.Println("Inserting ", N, " records in ", dbName)
 
 		db, err := newdb(dbName)
+		if err != nil {
+			panic(err)
```

Comment:
why not returning err and handling it upward rather than a panic?

Reply:
Since this is a simple program to generate DB files with 10K, 100K and 1M message records for perfomance testing, I think it's okay to just kill the program on failure. Error handling is not really necessary here (also all this code is running on main())